### PR TITLE
22847 - Amalgamation Application Output Display Type

### PR DESCRIPTION
**Pull request by ArwenQin**
*Issue #:* /bcgov/entity#22847

*Description of changes:*
- Updated the Amalgamation Application Output template, to display the type of Amalgamation Application (regular, short-form horizontal, short-form vertical)
- Adjust the font of Name of Amalgamated Company

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

**Comment by ArwenQin** [comment]
The type of Amalgamation Application
Before:
image

After:
image

And

image

**Comment by ArwenQin** [comment]
The Name of Amalgamated Company
Before:
image

After:
image

File: legal-api/report-templates/amalgamationApplication.html [hunk 1]
```diff
@@ -15,6 +15,13 @@
             [[logo.html]]
           </td>
           <td>
+            {% if amalgamationApplication.type == 'regular' %}
+              <div class="report-type">REGULAR FORM</div>
```
Comment (vysakh-menon-aot) [inline legal-api/report-templates/amalgamationApplication.html:19]: @severinbeauvais do you know if this has to be `REGULAR` or `REGULAR FORM`?

Comment (ArwenQin) [reply legal-api/report-templates/amalgamationApplication.html:19]: or REGULAR LONG FORM?

Comment (severinbeauvais) [reply legal-api/report-templates/amalgamationApplication.html:19]: That's a Mihai question.

Comment (severinbeauvais) [reply legal-api/report-templates/amalgamationApplication.html:19]: This is what I know:

image

**Review by severinbeauvais — APPROVED** [review]
Good job! This looks great!

I have the same question as Vysakh, which Mihai should answer.

File: legal-api/report-templates/amalgamationApplication.html (hunk 1 above)
Comment (ArwenQin) [reply legal-api/report-templates/amalgamationApplication.html:19]: Checking with Mihai now. Pending.

**Comment by ArwenQin** [comment]
@severinbeauvais 
Mihai Confirmed it should be REGULAR FORM (currently good)
image

Could you please merge?
Thank you.

File: legal-api/report-templates/amalgamationApplication.html (hunk 1 above)
Comment (ArwenQin) [reply legal-api/report-templates/amalgamationApplication.html:19]: Confirmed: REGULAR FORM

**Review by vysakh-menon-aot — APPROVED** [review]

**Pull request merged**